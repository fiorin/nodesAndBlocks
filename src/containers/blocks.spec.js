import React from "react";
import thunk from 'redux-thunk';
import { Provider } from "react-redux";
import configureMockStore from "redux-mock-store";
import ConnectedBlocks from "./Blocks";
import { create } from "react-test-renderer";

describe("<Blocks />", () => {
    const actions = {
        getBlocks: jest.fn()
    }

    const blocks = {
        list: [
            {
                id: 1,
                attributes: {
                    data: 'description'
                }
            }
        ]
    }

    it("should match snapshot", () => {
      const middlewares = [thunk];
      const store = configureMockStore(middlewares)({blocks});
      const component = create(
        <Provider store={store}>
          <ConnectedBlocks />
        </Provider>
      );
      const tree = component.toJSON();
  
      expect(tree).toMatchSnapshot();
    });
});
