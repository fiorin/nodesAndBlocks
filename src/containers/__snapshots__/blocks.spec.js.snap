// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Blocks /> should match snapshot 1`] = `
<div
  className="MuiBox-root MuiBox-root-1"
>
  <div
    className="MuiBox-root MuiBox-root-5 makeStyles-box-2"
  >
    <p
      className="MuiTypography-root makeStyles-id-3 MuiTypography-body1"
    >
      001
    </p>
    <p
      className="MuiTypography-root makeStyles-description-4 MuiTypography-body1"
    >
      description
    </p>
  </div>
</div>
`;
